includes/sanitizers/trait-amp-noscript-fallback.php

```diff
@@ -44,25 +44,25 @@ protected function initialize_noscript_allowed_attributes( $tag ) {
 	/**
 	 * Checks whether the given node is within an AMP-specific <noscript> element.
 	 *
-	 * @since 1.1
+	 * @param DOMNode $node DOM node to check.
 	 *
-	 * @param \DOMNode $node DOM node to check.
 	 * @return bool True if in an AMP noscript element, false otherwise.
+	 * @since 1.1
 	 */
-	protected function is_inside_amp_noscript( \DOMNode $node ) {
+	protected function is_inside_amp_noscript( DOMNode $node ) {
 		return 'noscript' === $node->parentNode->nodeName && $node->parentNode->parentNode && 'amp-' === substr( $node->parentNode->parentNode->nodeName, 0, 4 );
 	}
 
 	/**
 	 * Appends the given old node in a <noscript> element to the new node.
 	 *
-	 * @since 1.1
+	 * @param DOMNode     $new_node New node to append a noscript with the old node to.
+	 * @param DOMNode     $old_node Old node to append in a noscript.
+	 * @param DOMDocument $dom DOM document instance.
 	 *
-	 * @param \DOMNode     $new_node New node to append a noscript with the old node to.
-	 * @param \DOMNode     $old_node Old node to append in a noscript.
-	 * @param \DOMDocument $dom      DOM document instance.
+	 * @since 1.1
```

Comment:
Isn't `@since` supposed to be before `@param`? See https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods

```php
/**
 * Summary.
 *
 * Description.
 *
 * @since x.x.x
 *
 * @see Function/method/class relied on
 * @link URL
 * @global type $varname Description.
 * @global type $varname Description.
 *
 * @param type $var Description.
 * @param type $var Optional. Description. Default.
 * @return type Description.
 */
```

Reply:
Indeed. Not sure how these got mixed up here. Surprised that there isn't a sniff for that.